ui/src/views/infra/HostEnableDisable.vue

```diff
@@ -28,15 +28,15 @@
     >
       <a-alert type="warning">
         <template #message>
-          <span v-html="$t('message.confirm.enable.host')" />
+          <span v-html="resourcestate === 'Disabled' ? $t('message.confirm.enable.host') : $t('message.confirm.disable.host') " />
         </template>
       </a-alert>
-      <div v-show="enableKVMAutoEnableDisableSetting" class="reason">
+      <div v-show="kvmAutoEnableDisableSetting" class="reason">
         <a-form-item
           class="form__item"
           name="reason"
           ref="reason"
-          :label="'The setting \'enable.kvm.host.auto.enable.disable\' is enabled, ' +
+          :label="'The setting \'kvm.host.auto.enable.disable\' is enabled, ' +
```

Comment:
Maybe we can also rephrase this to something like: `The Auto Enable/Disable KVM Hosts functionality is enabled`

Reply:
Sorry to bother, I have just re-read the original issue and it seems to me we can fix it without needing to update the setting name but just updating this text and also the title as @shwstppr pointed out, leaving the setting as it originally was. What do you think @Pearl1594 @shwstppr ?

Reply:
Fine, I've reverted it.